docs/models.md

```diff
@@ -158,10 +155,14 @@ enum OrderStatus
 end
 
 class Order < Granite::Base
-  adapter mysql
-  table_name foos
+  connection mysql
+  table foos
 
   # Other fields
-  field status : OrderStatus, converter: Granite::Converters::Enum(OrderStatus, String) 
+  column status : OrderStatus, converter: Granite::Converters::Enum(OrderStatus, String) 
 end
 ```
+
+## Serialization
+
+Granite implements [JSON::Serializable](https://crystal-lang.org/api/JSON/Serializable.html) and [YAML::Serializable](https://crystal-lang.org/api/YAML/Serializable.html) by default.  As such, models can be serialized to/from JSON/YAML via the `#to_json`/`#to_yaml` and `.from_json`/`.from_yaml` methods.
```

Comment:
I could go either way.  On one hand it's nice to have there ready to go, but on the other if you're not going to use it it's basically just dead code.

@drujensen @robacarp WDYT?

Reply:
I personally don't use it.  I prefer using serializers.  However, I know this was requested several times so I'm ok leaving it since it seems like others prefer this approach.